src/MD5_fmt.c

```diff
@@ -28,7 +28,7 @@
 #endif
 
 #define FORMAT_LABEL			"md5crypt"
-#define FORMAT_NAME			"crypt(3) $1$"
+#define FORMAT_NAME			"crypt(3) $1$ (and variants)"
```

Comment:
Does this need a `benchmark-unify` update, or does `relbench` rely purely on `FORMAT_LABEL`? I don't recall.